src/main/java/com/cema/activity/entities/CemaWeighing.java

```diff
@@ -0,0 +1,116 @@
+package com.cema.activity.entities;
+
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.UUID;
+
+@Entity
+@Table(name = "weighing")
+public class CemaWeighing {
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(
+            name = "UUID",
+            strategy = "org.hibernate.id.UUIDGenerator"
+    )
+    @Column(name = "id")
+    private UUID id;
+
+    @Column(name = "category")
+    private String category;
+
+    @Column(name = "weight")
+    private Long weight;
+
+    @Column(name = "dental_notes")
+    private String dentalNotes;
+
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "execution_date")
+    private Date executionDate;
+
+    @Column(name = "establishment_cuig")
+    private String establishmentCuig;
+
+    @Column(name = "bovine_tag")
+    private String bovineTag;
+
+    public UUID getId() {
```

Comment:
cant we use loombok here?

Reply:
I'm not using lombok in entities to avoid issues with hibernate.